### ISSUE TITLE
Fail gracefully if locale C is not available

### ISSUE THREAD

**Issue by netravnen** [masonr/yet-another-bench-script#24]
Fail _gracefully_ if locale C is not available, can happen when it's a very fresh install of e.g. Debian. In my case, I set-up it up with default en_GB.UTF-8. Which causes the C locale to not be installed by default.

```
$ curl -s -k -g --header 'Host: raw.githubusercontent.com' https://\[2a04:4e42::133\]/masonr/yet-another-bench-script/master/yabs.sh | bash
bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
```

**Comment by netravnen** [comment]
I should mention the en_US.UTF-8 in the bash warning message was not configured by default either, when selecting during install en_GB.UtF-8 of Debian Buster.

**Comment by masonr** [comment]
Resolved in f93aa16

Added warning to output if C locale is not detected. Script will still run regardless in case the user is still using a locale that the script can interpret. Warning message will allow users to correct and re-run the test if they experience any issues.
